style.css

```diff
@@ -518,377 +514,373 @@ textarea:focus {
-    display: flex !important;
-    flex-direction: row !important;
-    text-align: start;
-    align-items: baseline;
-    justify-content: space-around;
-  }
-
-  form {
-    width: 50%;
-  }
-
-  .contact-p {
-    font-family: "Inter", sans-serif !important;
-    font-weight: 700 !important;
-    font-size: 40px !important;
-    width: 40%;
-  }
-
-  textarea {
-    font-size: 15px;
-    font-weight: 400;
-  }
-
-  .input__sec {
-    text-align: start;
-  }
-
-  .hr1 {
-    margin: 0;
-    margin-top: 40px;
-    align-self: stretch;
-    padding: 0;
-    margin-bottom: 17px;
-  }
-
-  .last-contact {
-    align-self: center;
-    justify-content: center;
-    margin-bottom: 25px;
-  }
-
-  .footer-contacts {
-    gap: 35px;
-    font-weight: 500;
-  }
-}
-
-@media (max-width: 990px) {
-  .hr1 {
-    margin: 0;
-    margin-top: 40px;
-    align-self: stretch;
-  }
-
-  .last-contact {
-    align-self: center;
-    justify-content: center;
-    margin-bottom: 15px;
   }
 
-  hr {
-    margin-bottom: 20px;
+  @media (max-width: 990px) {
+    .desk-head {
+      display: none;
+    }
+
+    .work-on {
+      align-self: center;
+    }
+  }
+
+  @media (min-width: 990px) {
+    .heading {
+      display: flex;
+      flex-direction: row-reverse;
+      justify-content: flex-end;
+      gap: 89px;
+      background-image: url(icons/Illustration-header@2x.png);
+      background-repeat: no-repeat;
+      background-position-y: -668px;
+      background-position-x: 510px;
+    }
+
+    .main-text {
+      display: flex;
+      flex-direction: column;
+      width: 485px;
+      margin-top: 65px;
+    }
+
+    .text {
+      font-family: "Crete Round", sans-serif;
+      font-size: 72px;
+      line-height: 81.5px;
+      line-height: 89%;
+      text-align: left;
+      vertical-align: top;
+      margin-bottom: 20px;
+    }
+
+    .text1 {
+      font-family: "Inter", sans-serif;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 32px;
+      line-height: 44px;
+      display: flex;
+      align-items: center;
+      color: #fff;
+      margin-bottom: 20px;
+    }
+
+    .text2 {
+      font-family: "Inter", sans-serif;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 20px;
+      line-height: 32px;
+      color: #f4f5f7;
+    }
+
+    .contacts {
+      display: flex;
+      flex-direction: column;
+      margin-top: 140px;
+    }
+  }
+
+  @media (min-width: 990px) {
+    .work-section {
+      display: flex;
+      flex-direction: column;
+      align-items: stretch;
+      padding: 100px 100px;
+      background-image: none;
+    }
+
+    .MY-works {
+      display: flex;
+      gap: 52px;
+      margin-bottom: 30px;
+    }
+
+    .part1 {
+      display: flex;
+      align-items: center;
+      gap: 20px;
+    }
+
+    h2 {
+      height: 55px;
+    }
+
+    .recent-work {
+      font-family: "Crete Round", sans-serif;
+      font-size: 40px;
+      text-align: left;
+      color: #091e42;
+    }
```

Comment:
- [ ] It would be better to use the same fonts that are provided in the Figma template to improve your UI design:

<img width="796" alt="image" src="https://user-images.githubusercontent.com/77190203/155486714-9152587f-2c1e-4cda-ae8c-e69e8ffbfd49.png">